.github/workflows/end-to-end.yml

```diff
@@ -13,3 +13,6 @@ jobs:
   juju-k8s-local:
     name: Test Juju, k8s-charm and local-auth
     uses: ./.github/workflows/e2e-juju-k8s-charm-local-auth.yml
+  jimm-machine:
+    name: Test JIMM machine charm
```

Comment:
Maybe this should mention OIDC as well to be consistent and also so that remember which runs are testing OIDC when looking through the list of jobs.